manifests/kube-applier.yaml

```diff
@@ -60,6 +60,8 @@ spec:
         app: kube-applier
     spec:
       serviceAccountName: kube-applier
+      securityContext:
+        fsGroup: 65533 # to make SSH key readable
```

Comment:
that will need a psp

Reply:
Not with the current roles, probably when we actually sort those.

https://trello.com/c/FzcjFU8N/601-kubernetes-role-for-kube-applier